test/k8s-integration/main.go

```diff
@@ -311,6 +314,27 @@ func handle() error {
 		if err != nil {
 			return fmt.Errorf("failed to install CSI Driver: %v", err)
 		}
+
+		// Dump all driver logs to the test artifacts
+		artifactsDir, ok := os.LookupEnv("ARTIFACTS")
+		if ok {
+			client, err := getKubeClient()
+			if err != nil {
+				return fmt.Errorf("failed to get kubeclient: %v", err)
+			}
+			out := podlogs.LogOutput{
+				StatusWriter:  os.Stdout,
+				LogPathPrefix: filepath.Join(artifactsDir, "pd-csi-driver") + "/",
+			}
+			ctx, cancel := context.WithCancel(context.Background())
+			defer func() {
+				cancel()
+			}()
+			if err = podlogs.CopyAllLogs(ctx, client, driverNamespace, out); err != nil {
+				return fmt.Errorf("failed to start pod logger: %v", err)
+			}
```

Comment:
1. Can we move this function to enable logging to a helper function?
2. For useGKEManagedDriver = true, the driver name space is not 'gce-pd-csi-driver'. Should we enable logging only if useGKEManagedDriver is false?

Reply:
1. Done
2. Yes, this is only called in the !useGKEManagedDriver block